Dockerfile

```diff
@@ -1,13 +1,6 @@
-FROM node:6
+FROM node:6.10.2
 
-RUN apt-get update
-RUN apt-get install apt-transport-https
-
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
-RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
-
-RUN apt-get update
-RUN apt-get install yarn
+RUN apt-get update && apt-get install apt-transport-https
```

Comment:
We should no longer need this line because apt-transport-https is purely for getting `yarn`.